src/config/passport.js

```diff
@@ -0,0 +1,21 @@
+import passport from 'passport';
+import GoogleStrategy from 'passport-google-oauth20';
+import FacebookStrategy from 'passport-facebook';
+import * as constant from './constants';
+
+passport.use(new GoogleStrategy({
+  callbackURL: constant.GOOGLE_CALLBACK,
+  clientID: constant.GOOGLE_CLIENT_ID,
```

Comment:
You have done good work bro, but why don't you consider  abstracting the relevant strategies and then import the abstracted object into their respective passport.use

Reply:
Ok, I get it. thanks for that.